js/package.json

```diff
@@ -2,13 +2,16 @@
   "name": "@flarum/flarum-ext-emoji",
   "version": "0.0.0",
   "dependencies": {
+    "emojibase-data": "^3.0.0",
     "flarum-webpack-config": "^0.1.0-beta.8",
     "textarea-caret": "^3.1.0",
+    "twemoji": "^11.0.0",
     "webpack": "^4.0.0",
     "webpack-cli": "^3.0.7"
   },
   "scripts": {
-    "build": "webpack --mode production",
-    "watch": "webpack --mode development --watch"
+    "build": "npm run generate-emoji-map; webpack --mode production",
+    "watch": "npm run generate-emoji-map; webpack --mode development --watch",
+    "generate-emoji-map": "node ./scripts/generate-emoji-map.js"
```

Comment:
Could we use `prebuild` and `prewatch` here, so that we are more in line with other extensions?